setup.py

```diff
@@ -335,34 +342,22 @@ def run_install_headers(self):
     for e in extensions:
         e.define_macros.append(('PG_ENABLE_ARM_NEON', '1'))
 
-#decide whether or not to enable new buffer protocol support
-enable_newbuf = False
-if sys.version_info >= (2, 6, 0):
-    try:
-        sys.pypy_version_info
-    except AttributeError:
-        enable_newbuf = True
+# decide whether or not to enable new buffer protocol support (PEP 3118)
+# old CPython versions without newbuf are no longer supported!
+# new PYPY also supports PEP 3118
+enable_newbuf = True
 
-if enable_newbuf:
-    enable_newbuf_value = '1'
-else:
-    enable_newbuf_value = '0'
+# TODO: remove all PG_ENABLE_NEWBUF conditionals from C code
```

Comment:
I'd like to move towards leaving TODO notes as github issues instead of leaving them in the code as part of the (py)lint cleanups.